packages/cli/src/rpc/explain/collect-context.ts

```diff
@@ -0,0 +1,136 @@
+import { ContextV2 } from '@appland/navie';
```

Comment:
This file is essentially `buildContext`, but with a more (IMO) clear name, since the job of the function is not just to build the context, but to collect it from the available assets.